code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm

```diff
@@ -67,14 +72,27 @@
 /datum/rune_spell/proc/midcast(var/mob/add_cultist)
 	return
 
+/datum/rune_spell/proc/blood_pay()
+	var/data = use_available_blood(activator, cost_invoke)
+	if (data["result"] == "failure")
```

Comment:
I can see this going weird in the future. Put "result" and "failure" and the like from use_available_blood as defines.